### ISSUE TITLE
Feature request: a function to load file content as string

### ISSUE THREAD

**Issue by begoon** [casey/just#2508]
For example:

```justfile
TAG := trim(file_content("TAG.txt"))

docker-push:
	docker push ...:{{VERSION}}
```

Proposed PR: https://github.com/casey/just/pull/2507

**Comment by laniakea64** [comment]
Why not use `cat`? -
```
TAG := trim(`cat TAG.txt`)

docker-push:
	docker push ...:{{VERSION}}
```

**Comment by begoon** [comment]
It is possible to use `cat`. However, loading a file is an everyday use case in CI scripting, so having a fast built-in would be convenient. It is similar to the `datetime()` function, which can be replaced by `(shell date ...)`, but there is a built-in.

**Comment by begoon** [comment]
Also, `cat` may not be available on Windows.

**Comment by casey** [comment]
Merged in #2507 as `read_to_string(path)`. Perhaps we should change it to just `read(path)` before the next release? I originally intended the name to be more verbose to be more explicit, but in reality, `just` only has strings, so there's no way the function can return anything else.

**Comment by casey** [comment]
Renamed to `read(path)` in #2518.
